### ISSUE TITLE
Fix minimum allowed term (critical/warning) duration 

### ISSUE THREAD

**Issue by dvishniakov** [newrelic/terraform-provider-newrelic#889]
Hi there,

### Terraform Version
0.12, 0.13

### Affected Resource(s)
- newrelic_nrql_alert_condition

### Terraform Configuration Files
This is simplified and interpolated part of the config:
```hcl
resource "newrelic_nrql_alert_condition" "nrql_cond" {
....
  critical {
    ....
    threshold_duration    = 60
  }

```

### Debug Output
> Error: "critical.0.threshold_duration" must be between 120 and 7200 inclusive, got: 60

### Expected Behavior
Allow to set 60 seconds (1 minute) as a duration for NRQL alert, because it is allowed through UI.
Provided screenshot shows that after manual modification in NR UI to 1 minute, plugin correctly receives the value of 60s and tries to get it back to 2 minutes: ![Screen Shot 2020-09-17 at 3 30 26 PM](https://user-images.githubusercontent.com/11319948/93518987-9ba54200-f8fb-11ea-93dd-f8e5b03074f4.png)

### Actual Behavior
plugin doesn't allow it

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### References
-Resource attribute with incorrect (outdated?) requirement of 2 minutes or more: https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition#threshold
- Documentation mentioning 1 minute for NRQL alert (unfold Set Time Intervals): https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition:
> and others have a minimum time interval of 1 minute (for example, NRQL alert conditions).

**Comment by sanderblue** [comment]
Hi @dvishniakov thanks for reporting. It appears something must've changed in an upstream API. Can we see more of your `newrelic_nrql_alert_condition` configuration block (sensitive info redacted) in order to do some more debugging? Different types of NRQL conditions have different validation criteria for threshold duration so we want to make sure we're handling things correctly for the different types. Much appreciated 😎

**Comment by sanderblue** [comment]
@dvishniakov Hi there! We released a fix in [**v2.7.4**](https://github.com/newrelic/terraform-provider-newrelic/releases/tag/v2.7.4). Please let us know if this fixes the issue for you. Thanks!
